google-mysql.yml

```diff
@@ -26,40 +26,35 @@ plan_updateable: true
 plans:
 - name: small
   id: 8809fe67-99b5-48dd-a6dd-890ee45e86be
-  description: 'Beta - MySQL v5.7, minimum 2 cores, minimum 4GB ram, 10GB storage'
+  description: 'Beta - MySQL v5.7, 2 cores, 8GB ram, 10GB storage'
   display_name: "small (Beta)"
   properties:
-    cores: 2
     mysql_version: "MYSQL_5_7"
     storage_gb: 10
+    tier: db-n1-standard-2
 - name: medium
   id: 753accb8-52a7-4673-8216-ffc539351aea
-  description: 'Beta - MySQL v5.7, minimum 4 cores, minimum 8GB ram, 10GB storage'
+  description: 'Beta - MySQL v5.7, 4 cores, 16GB ram, 10GB storage'
   display_name: "medium"
   properties:
-    cores: 4
     mysql_version: "MYSQL_5_7"
     storage_gb: 10
+    tier: db-n1-standard-4
 - name: large
   id: 8f8fc4ab-2b37-41da-bc6a-e915497dac33
-  description: 'Beta - MySQL v5.7, minimum 8 cores, minimum 16GB ram, 20GB storage'
+  description: 'Beta - MySQL v5.7, 8 cores, 32GB ram, 20GB storage'
   display_name: "large"
   properties:
-    cores: 8
     mysql_version: "MYSQL_5_7"
     storage_gb: 20
+    tier: db-n1-standard-8
 provision:
   plan_inputs: []
   user_inputs:
-  - field_name: cores
+  - field_name: tier
     required: true
-    type: number
-    details: Minimum number of cores for service instance.
-    default: 2
-    constraints:
```

Comment:
**suggestion**: We could add a constraint to check if the value matches the GCP name constraint

Reply:
Thank you very much for your comment @jhvhs. I will add the constraint now!